fury/lib.py

```diff
@@ -1,14 +1,42 @@
+from typing import TypeAlias
+
 import pygfx as gfx
+from wgpu.gui.auto import WgpuCanvas, run
+from wgpu.gui.offscreen import WgpuCanvas as OffscreenWgpuCanvas
+
+from fury.optpkg import optional_package
+
+jupyter_pckg_msg = (
+    "You do not have jupyter-rfb installed. The jupyter widget will not work for "
+    "you. Please install or upgrade jupyter-rfb using pip install -U jupyter-rfb"
+)
+
+jupyter_rfb, have_jupyter_rfb, _ = optional_package(
+    "jupyter-rfb", trip_msg=jupyter_pckg_msg
+)
+
+if have_jupyter_rfb:
+    from wgpu.gui.jupyter import WgpuCanvas as JupyterWgpuCanvas
 
 Texture = gfx.Texture
 AmbientLight = gfx.AmbientLight
 Background = gfx.Background
 BackgroundSkyboxMaterial = gfx.BackgroundSkyboxMaterial
-Camera = gfx.Camera
-Controller = gfx.Controller
+
+# Classes that needed to be written as types
+Camera: TypeAlias = gfx.Camera
+Controller: TypeAlias = gfx.Controller
+Scene: TypeAlias = gfx.Scene
+Viewport: TypeAlias = gfx.Viewport
```

Comment:
What do you need the TypeAlias here ?

Reply:
It gives pre-commit error as these classes are used to either extend the class or used as type in a dataclass.

Reply:
strange, ok, I look into it later, merging

Reply:
@skoudoro  you can take a look at this. The pre-commit points to this resource when it gives the error. https://mypy.readthedocs.io/en/stable/common_issues.html#variables-vs-type-aliases